src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+     */
+    private static void getMessage() {
+        Scanner in = new Scanner(System.in);
+        String[] input = new String[MAX_NO];
+        int i = 0;
+
+        while (true) {
+            input[i] = in.nextLine();
+            if (input[i].equals("bye")) {
+                printBye();
+                break;
+            } else if (input[i].equals("list")) {
+                listTasks(tasks);
+            } else if (input[i].contains("done")) {
+                markAsDone(input[i]);
+            } else if (input[i].contains("deadline")){
+                addDeadline(input[i]);
+            } else if (input[i].contains("event")) {
+                addEvent(input[i]);
+            } else if (input[i].contains("todo")) {
+                addToDo(input[i]);
+            } else {
+                echoMessage(input[i]);
+            }
+            i++;
+        }
+    }
+
+    public static void printGreeting() {
+        String logo = "      /-\\    /-\\ \n" //6 spaces
+            + "     /  |_9_/  |\n" //5 spaces
+            + "    /,,o  3  o,,\\ \n"; //4 spaces
+        printDivider();
+        System.out.println("\tHullo I'm Toto!\n" + logo); //4 spaces
+        System.out.println("\tHow can Toto help today?"); //4 spaces
+        printDivider();
+    }
+
+    public static void printDivider(){
+        System.out.println("-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-");
+    }
+
+    public static void printBye(){
+        String logo = "      /-\\    /-\\ \n"
+            + "     /  |_9_/  |\n"
+            + "    / TT  w  TT \\ \n";
+        printDivider();
+        System.out.println("\tToto will be lonely... :<\n" + logo);
+        printDivider();
+    }
+
+    public static void listTasks (Task[] tasks) {
+        printDivider();
+        if (numOfTasks == 0) {
+            System.out.println("\tyour task list is empty");
+            printDivider();
+        } else {
+            for (int j = 0; j < numOfTasks; j++) {
+            System.out.println((j+1) + ": " + tasks[j]);
+            }
+        }
+        printDivider();
+    }
+
+    public static void addDeadline(String input) {
+        numOfTasks++;
+        input = input.replace("deadline", " ").trim();
+        int dividerPosition = input.indexOf("/by");
+        String by = input.substring(dividerPosition+3).trim();
+        String taskDescription = input.substring(0, dividerPosition).trim();
+        tasks[numOfTasks-1] = new Deadline(taskDescription, by);
+        printDivider();
+        System.out.println("\tadded: " + taskDescription);
+        System.out.println(numOfTasks + ":" + tasks[numOfTasks-1]);
+        System.out.println("\tnow you have " + numOfTasks + " task(s)");
+        printDivider();
+    }
+
+    private static void addEvent(String input) {
+        numOfTasks++;
+        input = input.replace("event", " ");
+        int dividerPosition = input.indexOf("/at");
+        String time = input.substring(dividerPosition+3).trim();
```

Comment:
Consider putting a white space before and after the "+" for better readability, similarly for the rest of the code

Reply:
hmm, not too sure about this, but this website:
https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html
states that "Blank spaces should never separate unary operators such as unary minus, increment ("++"), and decrement ("--") from their operands."